### ISSUE TITLE
Display Settings For Correct Answer

### ISSUE THREAD

**Issue by Brtelfer** [memphis-iis/mofacts#1393]
Is there a way through the uiSettings to make the extra feedback for the correct answer not display? It would be nice to have it only say "correct" when the answer is correct and the additional details when the answer is incorrect.
<img width="800" alt="image" src="https://github.com/memphis-iis/mofacts/assets/106611498/cd24ccfc-8c32-4d56-b6e6-eb6782cf0152">

**Comment by imrryr** [comment]
So, there are two small issues here. It seems impossible to get the feedback only once. Can we block the second time (I'm not sure it is ever appropriate to present it twice, but maybe I don't see the use case...)? Also, can we block the correct answer entirely, just have it say "Correct."? Thanks! @JRustyHaner

**Comment by JRustyHaner** [comment]
@imrryr which package? I'll need to investigate

**Comment by imrryr** [comment]
Well all of my quiz packages behave this way, so you could look at those. This is especially an issue for feedback on multiple choice. Not sure I noticed an issue for the fill-in trials, but we want the same behavior to be possible there too.
[unit1.zip](https://github.com/memphis-iis/mofacts/files/14166547/unit1.zip)

**Comment by JRustyHaner** [comment]
@imrryr  with the propsed UI settings in the PR #1396, you're settings would be modified to

"displayUserAnswerInFeedback": "onIncorrect",
"onlyShowSimpleFeedback": "onCorrect",

which are the defaults, so you can either set these, or remove the lines from your settings entirely.
